app/experimenter/nimbus-ui/src/components/PageResults/TableMetricConversion/index.tsx

```diff
@@ -89,7 +89,13 @@ const TableMetricConversion = ({
                       results={results[branch]}
                       group={GROUP.OTHER}
                       tableLabel={TABLE_LABEL.PRIMARY_METRICS}
-                      {...{ metricKey, displayType, branchComparison, bounds }}
+                      // isControlBranch={branch === controlBranchName}
```

Comment:
@emtwo it doesn't look like we're actually using this table anywhere. Should we remove it or am I missing something (maybe we plan to use it again later)? 😬

Reply:
Initially we were using it for primary metrics but it was a bit hard coded. This used to be called `TableMetricPrimary` but it only had conversion metrics. However, primary metrics are not always conversion metrics and now primary metrics are defaulting to `TableMetricCount`. 

In the future, Jetstream would specify whether a primary or secondary metric is a count or conversion metric (or something else) and that's how we would decide what table to use.Right now there is no logic in there for making that decision because it's not part of Jetstream yet either

Reply:
I'll include this in the Context refactor then since it sounds like we may need to use it later, thanks. 😄